api/users/permissions.py

```diff
@@ -0,0 +1,15 @@
+from website.models import OAuth2App
+from rest_framework import permissions
+
+
+class OwnerOnly(permissions.BasePermission):
```

Comment:
@sloria, I wonder if it would make sense to consolidate all permissions to api.base.permissions. Prooobably not, but the reason I'm thinking so is if we do a refactoring of permissions, we would have them all in the same place and it'll be less likely to miss something. Also, permissions seem more like a central thing than a collection of things associated with the objects. Not a strong feeling, but a little niggling one, so I thought I'd see what you thought.